src/odemis/util/filename.py

```diff
@@ -30,16 +30,16 @@
 from datetime import datetime, timedelta
 
 
-def guess_pattern(fn):
+def guess_pattern(fn, is_count=True):
```

Comment:
`is_count` doesn't sound proper English and make it a little unclear. I'd suggest `detect_count`.